docs/tutorial.rst

```diff
@@ -100,7 +100,69 @@ large sections of the tree which do not have any variable sites using
 
 .. _troubleshooting:
 
+*******************************************************
+Inferring and Dating Tree Sequences with Ancient Samples
+*******************************************************
 
+``tsdate`` and ``tsinfer`` can be used together to infer tree sequences from both
+modern and ancient samples. The following recipe shows how this is accomplished
+with a few lines of Python.
+
+.. code-block:: python
+
+   import msprime
+   import tsdate
+   import tsinfer
+   import tskit
+
+   import numpy as np
+
+
+   def historical_samples_example():
```

Comment:
Good idea!
I'll call it `make_historical_samples` to avoid confusion with "getting" an attribute

Reply:
I guess you might also want to put all the other (pipeline) steps into a single function in the example, so that people can simply copy and paste it for their own use?